docs/helpers/extended_vars.yaml

```
@@ -145,7 +145,7 @@ variables:
     section in the documentation for more details on defaults.
   do_ignore: false
 - rawname: RUN_CMD_TEST
```

I think this should be ignored in the documentation.
```suggestion
- rawname: RUN_CMD_TEST
  do_ignore: true
```